currency-converter/currency.js

```diff
@@ -0,0 +1,75 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const rates = {
+    timestamp: Date.now(),
+    base: "EUR",
+    date: new Date().toISOString().split("T")[0],
+    rates: {
+      USD: 1.23,
+      GBP: 0.88,
+      EUR: 1.82,
+    },
+  };
+
+  const insertRateForm = document.getElementById("insertRateForm");
+  const convertForm = document.getElementById("convertForm");
+  const updateRateForm = document.getElementById("updateRateForm");
+  const conversionResult = document.getElementById("conversionResult");
+
+  insertRateForm.addEventListener("submit", (event) => {
+    event.preventDefault();
+    const baseCurrency = document
+      .getElementById("baseCurrency")
+      .value.toUpperCase();
+    const targetCurrency = document
+      .getElementById("targetCurrency")
+      .value.toUpperCase();
+    const rate = parseFloat(document.getElementById("rate").value);
+
+    rates.base = baseCurrency;
+    rates.rates[targetCurrency] = rate;
+
+    console.log("Inserted new rate:", rates);
+    alert(`Inserted new rate: 1 ${baseCurrency} = ${rate} ${targetCurrency}`);
```

Comment:
**Optional comment**
You could display the confirmation message on the UI instead of using an `alert`. It's less aggressive for the user experience.

Reply:
Thank you for the the suggestion and correction.